pkg/cluster_agent/main.py

```diff
@@ -6,41 +6,59 @@
 from os import getenv
 from datetime import datetime, timezone
 from traceback import format_exc
-from kubernetes import client, config
-from cluster_scanner import ClusterScanner
+from kubernetes import config, client
+from cluster_scanner import ClusterScanner, ResourceScanResult
+from resource_sender import ResourceSender
 
 SCAN_INTERVAL_SECONDS = 60
 EPSAGON_TOKEN = getenv("EPSAGON_TOKEN")
-CLUSTER_NAME = getenv("EPSAGON_CLUSTER_NAME")
+DEFAULT_CLUSTER_NAME = "K8s Cluster"
```

Comment:
Do we want to allow spaces in the cluster name? (just need to make sure to handle that properly in the backend)

Reply:
you're right, will separate it with "-". We will def test this case